pkg/postage/postagecontract/contract.go

```diff
@@ -0,0 +1,187 @@
+package postagecontract
```

Comment:
code copy missing

Reply:
also, i think @zelig would probably want to call this package `subscription` or something like that

Reply:
not this one, no